src/v/cloud_storage/remote_segment.cc

```diff
@@ -294,7 +298,11 @@ ss::future<> remote_segment::do_hydrate() {
 ss::future<> remote_segment::maybe_materialize_index() {
     ss::gate::holder guard(_gate);
     auto path = _path().native() + ".index";
-    offset_index ix(_base_rp_offset, _base_rp_offset - _base_offset_delta, 0);
+    offset_index ix(
+      _base_rp_offset,
+      _base_rp_offset - _base_offset_delta,
+      0,
+      remote_segment_sampling_step_bytes);
```

Comment:
is `remote_segment_sampling_step_bytes` here used as the default sampling step when constructing the index?

Reply:
yes